src/main/resources/application.yml

```
@@ -21,6 +21,47 @@ spring:
           maxConnectionIdleTimeMS: ${MONGODB_CONNECTIONPOOL_MAX_CONNECTION_IDLE_MS:120000}
           maxConnecting: ${MONGODB_CONNECTIONPOOL_MAX_CONNECTING:2}
 
+  cloud:
+    stream:
+      binders:
+        notification-outcome:
```

let's use a more significant name (fix each reference to this value)
```suggestion
        transaction-outcome:
```